core/src/main/java/org/apache/accumulo/core/spi/balancer/data/TabletStatistics.java

```diff
@@ -28,6 +28,7 @@ public interface TabletStatistics extends Comparable<TabletStatistics> {
 
   long getNumEntries();
 
+  @Deprecated(since = "3.1")
```

Comment:
Could add the forRemoval parameter to the deprecation annotation, it defaults to false.
```suggestion
  @Deprecated(since = "3.1", forRemoval=true)
```